index.html

```diff
@@ -203,6 +203,13 @@ <h2 class="component-header-box text-blue">Gear Up for Your Adventure</h2>
   </div>
 </section>
 
+<section>
```

Comment:
I think this `<section/>` needs to be removed looks like something you were testing?

![screenshot 2020-06-22 at 15 53 46](https://user-images.githubusercontent.com/56852436/85329476-9a9d7980-b4a0-11ea-85ef-8d3594b3bba1.png)